From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeff Long <willcode4@gmail.com>
Date: Thu, 14 Oct 2021 10:42:11 -0400
Subject: [PATCH] grc: typo in Connection.py

default for dict is just an extra arg, no default= keyword

Signed-off-by: Jeff Long <willcode4@gmail.com>
(cherry picked from commit 553248575079794236923ad3aa03cda03ed902d0)
Signed-off-by: Jeff Long <willcode4@gmail.com>
(cherry picked from commit 39a8c4a36e1b354943dd1e2e818446a63065abce)
Signed-off-by: Jeff Long <willcode4@gmail.com>
---
 grc/core/Connection.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grc/core/Connection.py b/grc/core/Connection.py
index 1501d3897..8a4863324 100644
--- a/grc/core/Connection.py
+++ b/grc/core/Connection.py
@@ -106,7 +106,7 @@ class Connection(Element):
         source_dtype = self.source_port.dtype
         sink_dtype = self.sink_port.dtype
         if source_dtype != sink_dtype and source_dtype not in ALIASES_OF.get(
-            sink_dtype, default=set()
+            sink_dtype, set()
         ):
             self.add_error_message('Source IO type "{}" does not match sink IO type "{}".'.format(source_dtype, sink_dtype))
 
-- 
2.32.0

